Olympus.FNA/UI/Elements/TextInput.cs

```diff
@@ -208,8 +209,10 @@ private enum Action {
         const char HOME = (char)0x02;
         const char END = (char)0x03;
         
-        // Ignore enter (at least for now)
-        if (chr == '\n' || chr == '\r') return;
+        if (chr == '\n' || chr == '\r') {
+            ConfirmCallback?.Invoke(this);
+            return;
+        }
```

Comment:
Quick nitpick: only skip newline characters if the callback was set, and if it returns false, otherwise allow it.
Because with whats currently here the user has no way of inputting a newline if its allowed